src/BitStringAddresses/multicomponent.jl

```diff
@@ -132,26 +132,53 @@ end
     FermiFS2C(onr_a, onr_b)
 
 Fock state address with two fermionic (spin) components. Alias for [`CompositeFS`](@ref)
-with two [`FermiFS`](@ref) components.
-Construct by specifying either two compatible [`FermiFS`](@ref)s or [`onr`](@ref)s.
+with two [`FermiFS`](@ref) components. Construct by specifying either two compatible
+[`FermiFS`](@ref)s, two [`onr`](@ref)s, or the number of modes followed by `mode =>
+occupation_number` pairs, where negative occupation numbers will go to the second
+component. See examples below.
```

Comment:
This is clearer. I say this because if something like `mode_num => -2` is attempted then the error message is
`ArgumentError: Invalid ONR: may only contain 0s and 1s.`
While this is true, it is a bit confusing for `FermiFS2C`.

```suggestion
with two [`FermiFS`](@ref) components. Construct by specifying either two compatible
[`FermiFS`](@ref)s, two [`onr`](@ref)s, or the number of modes followed by `mode =>
occupation_number` pairs, where `occupation_number=1` will put a particle in the first 
component and `occupation_number=-1` will put a particle in the second component.
See examples below.
```